### PR TITLE
Update tcp filter report interval config to use google.protobuf.Duration type.

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Make this change according to review comment in #328

### CODE REVIEW

File: mixer/v1/config/client/client_config.proto [hunk 1]
```diff
@@ -125,8 +126,8 @@ message TcpClientConfig {
...
   QuotaSpec connection_quota_spec = 5;
 
-  // Specify report interval in milliseconds to send periodical reports for
-  // long TCP connections. If not specified, or 0 protobuf default value,
-  // the interval is 10 seconds.
-  int32 report_interval_ms = 6;
+  // Specify report interval to send periodical reports for long TCP
+  // connections. If not specified, or 0 protobuf default value, the interval
+  // is 10 seconds.
+  google.protobuf.Duration report_interval = 6;
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: Could you also change stats_interval in line 82?

File: mixer/v1/config/client/client_config.proto [hunk 2]
```diff
@@ -125,8 +126,8 @@ message TcpClientConfig {
   // It applies on the new TCP connections.
   QuotaSpec connection_quota_spec = 5;
 
-  // Specify report interval in milliseconds to send periodical reports for
-  // long TCP connections. If not specified, or 0 protobuf default value,
-  // the interval is 10 seconds.
-  int32 report_interval_ms = 6;
+  // Specify report interval to send periodical reports for long TCP
+  // connections. If not specified, or 0 protobuf default value, the interval
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: remove 
0 protobuf default value.  
Now duration is a message,

File: mixer/v1/config/client/client_config.proto (hunk 1 above)
Comment (JimmyCYJ) [reply mixer/v1/config/client/client_config.proto]: Done. Thanks!

File: mixer/v1/config/client/client_config.proto [hunk 3]
```diff
@@ -76,10 +76,10 @@ message TransportConfig {
   // Specifies the policy when failed to connect to Mixer server.
   NetworkFailPolicy network_fail_policy = 4;
 
-  // Specify refresh interval in milliseconds to write mixer client statistics
-  // to Envoy share memory. If not specified, or 0 protobuf default value, the
-  // interval is 10 seconds
-  int32 stats_update_interval_ms = 5;
+  // Specify refresh interval to write mixer client statistics to Envoy share
+  // memory. If not specified, or 0 protobuf default value, the interval is 10
+  // seconds.
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: remove "0 protobuf default value,".  It doesn't apply any more and Duration is a message

**Pull request merged**